pywemo/discovery.py

```diff
@@ -93,15 +93,27 @@ def device_from_uuid_and_location(uuid, mac, location,
     if uuid.startswith('uuid:Humidifier'):
         return Humidifier(url=location, mac=mac,
                           rediscovery_enabled=rediscovery_enabled)
+    if uuid.startswith('uuid:'):
+        try:
+            # unknown device, try to make a basic object to allow manual
+            # interaction
+            return UnknownDevice(url=location, mac=mac,
```

Comment:
Should this also log something, along with some info about the device (eg `unsupported device X found - returning  Unknown Device`).

Reply:
I've added a short log here, but see further discussion below.